lua/onenord/theme.lua

```diff
@@ -551,7 +551,14 @@ local function load_plugins(config)
     CmpItemAbbrDeprecated = { fg = onenord.fg },
     CmpItemAbbrMatch = { fg = onenord.blue, style = "bold" },
     CmpItemAbbrMatchFuzzy = { fg = onenord.blue, underline = true },
-    CmpItemKind = { fg = onenord.purple },
+    CmpItemKindVariable = { fg = onenord.cyan },
+    CmpItemKindInterface = { fg = onenord.cyan },
+    CmpItemKindText = { fg = onenord.cyan },
+    CmpItemKindFunction = { fg = onenord.pink },
+    CmpItemKindMethod = { fg = onenord.pink },
+    CmpItemKindKeyword = { fg = onenord.fg },
+    CmpItemKindProperty = { fg = onenord.yellow },
+    CmpItemKindUnit = { fg = onenord.yellow },
     CmpItemMenu = { fg = onenord.light_gray },
```

Comment:
What do you think of something like this? I tried to relate the colours to the treesitter highlights but let me know if anything doesn't look right or you have a preference for something else.

```suggestion
    CmpItemMenu = { fg = onenord.light_gray },

    CmpItemKindText = { fg = onenord.orange },
    CmpItemKindMethod = { fg = onenord.blue },
    CmpItemKindFunction = { fg = onenord.blue },
    CmpItemKindConstructor = { fg = onenord.yellow },
    CmpItemKindField = { fg = onenord.blue },
    CmpItemKindClass = { fg = onenord.yellow },
    CmpItemKindInterface = { fg = onenord.yellow },
    CmpItemKindModule = { fg = onenord.blue },
    CmpItemKindProperty = { fg = onenord.blue },
    CmpItemKindValue = { fg = onenord.orange },
    CmpItemKindEnum = { fg = onenord.yellow },
    CmpItemKindKeyword = { fg = onenord.purple },
    CmpItemKindSnippet = { fg = onenord.green },
    CmpItemKindFile = { fg = onenord.blue },
    CmpItemKindEnumMember = { fg = onenord.cyan },
    CmpItemKindConstant = { fg = onenord.orange },
    CmpItemKindStruct = { fg = onenord.yellow },
    CmpItemKindTypeParameter = { fg = onenord.yellow },
```

Reply:
Thanks, looks great! Good idea to relate it to the treesitter highlights.